pages/data-design/start-using-inspector.mdx

```diff
@@ -15,23 +37,69 @@ After <Link passHref href="/workspace/inspector#configuring-sources"><a>setting
   />
 </center>
 
-<Link passHref href="/workspace/inspector">Inspector tab description</Link>  
+<Link passHref href="/workspace/inspector"><a>Inspector tab description</a></Link>
 <br/>
 <br/>
 
-Open the <Link passHref href="/workspace/inspector#Overview">Inspector overview</Link>, browse the list or search for specific event, select it and the right panel will appear with detailed information about the health of the given event.
+Open the <Link passHref href="/workspace/inspector#Overview"><a>Inspector overview</a></Link>, browse the list or search for a specific event. Select an event and detailed information about the health of the given event will appear in the panel on the right.
```

Comment:
It looks like the images are messing up the location of the anchor (#) in the link. It's probably because they don't have a height associated with them so they get calculated after the anchor locating and you end up above the actual destination. I'm not sure if we can do anything about that now.

Reply:
Good point, I was always wondering why is it happening